src/app/core/models/show-style-variant.ts

```diff
@@ -0,0 +1,28 @@
+export interface ShowStyleVariantBlueprintConfiguration {
+    SelectedGfxSetupName: {
+        value: string
+        label: string
+    }
+    GfxDefaults: {
+        id?: string
+        DefaultSetupName: {
+            value: string
+            label: string
+        },
+        DefaultSchema: {
+            value: string
+            label: string
+        },
+        DefaultDesign: {
+            value: string
+            label: string,
+        }
+    }[]
```

Comment:
Bonus info: The reason it's saved as an array is because of the "framework" Core provides to make the Settings page for Blurprints which forced us to use a "Table", which makes the entries an array.

Reply:
> Sidenote: We want to get rid of the Gfx and instead write Graphic. 

I have been enlightened